internal/core/context.go

```diff
@@ -26,14 +27,14 @@ type meta struct {
 	OverrideEnv  map[string]string
 	OverrideExec OverrideExecFunc
 	CliConfig    *cliConfig.Config
+	Platform     platform.Platform
```

Comment:
Can/Should we change the user agent if the CLI is doing a request in the webassembly mode ? How can we track usage for it?